Formula/astrometry-net.rb

```diff
@@ -27,35 +27,53 @@ class AstrometryNet < Formula
   depends_on "cairo"
   depends_on "cfitsio"
   depends_on "gsl"
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
   depends_on "libpng"
   depends_on "netpbm"
   depends_on "numpy"
-  depends_on "python@3.9"
+  depends_on "python@3.10"
   depends_on "wcslib"
 
   resource "fitsio" do
     url "https://files.pythonhosted.org/packages/23/ec/280f91842d5aeaa1a95dc1d86d64d3fe57a5a37a98bb39b73a963f5dc91d/fitsio-1.1.6.tar.gz"
     sha256 "3e7e5d4fc025d8b6328ae330e72628b92784d4c2bb2f1f0caeb75e730b2f91a5"
   end
 
+  # Three patches to fix building with numpy 1.23+
+  # https://github.com/dstndstn/astrometry.net/issues/262
+  patch do
+    url "https://github.com/dstndstn/astrometry.net/commit/ae47640f8abe0593a2a757c2654201e650460445.patch?full_index=1"
+    sha256 "41e245ef699cfd7f04199e678942071822eb5bea071b6d11e82013d4af703c1b"
+  end
+
+  patch do
+    url "https://github.com/dstndstn/astrometry.net/commit/816976ae7e15fd1b9c589312a7aadc802f6915fe.patch?full_index=1"
+    sha256 "02bfe7523791672896e7ec9a16e2c0968149d9f1dc79c37803748d60da123ac9"
+  end
+
+  patch do
+    url "https://github.com/dstndstn/astrometry.net/commit/943c61dc42fb9a57167fab4e4fd7a88f058ab3fc.patch?full_index=1"
+    sha256 "6034b689a172f27a95a7fb10c718005c862ad3d84c44a41ce191c9c82fee850d"
+  end
```

Comment:
Patches can be dropped after https://github.com/Homebrew/homebrew-core/pull/107255.